tests/functional/disaster-recovery/sc_arbiter/test_mon_osd_failures.py

```diff
@@ -136,8 +136,47 @@ def finalizer():
     request.addfinalizer(finalizer)
 
 
+@pytest.fixture(scope="class")
+def setup_cnv_workload(request, cnv_workload, setup_cnv):
+
+    logger.info("Setting up CNV workload and creating some data")
+    vm_obj = cnv_workload(volume_interface=constants.VM_VOLUME_PVC)[0]
+    vm_obj.run_ssh_cmd(command="dd if=/dev/zero of=/file_1.txt bs=1024 count=102400")
+    md5sum_before = vm_obj.run_ssh_cmd(command="md5sum /file_1.txt")
+
+    def finalizer():
+
+        # check vm data written before the failure for integrity
+        logger.info("Waiting for VM SSH connectivity!")
+        vm_obj.wait_for_ssh_connectivity()
+        md5sum_after = vm_obj.run_ssh_cmd(command="md5sum /file_1.txt")
+        assert (
+            md5sum_before == md5sum_after
```

Comment:
shouldn't this be checked in the test case?

Reply:
This class scoped fixture is only developed for this particular use case. The reason is because we want to perform workload verification all the tests within Test class is executed. There wont be workload specific verification after each test is executed. And only possible way for me to do that is through a separate class fixture that will setup before all the tests are executed and workload verification is done after all the tests are completed

Reply:
As discussed, please move it to the test case